week2_and_week3/Singleton/singleton.py

```diff
@@ -0,0 +1,31 @@
+from typing import Dict
+
+class Singleton:
```

Comment:
no to nie jest singleton. W singletonie chodzi o to, że pozwalasz na stworzenie tylko **_jednej_** instancji klasy.